libs/nss.c

```diff
@@ -108,8 +120,15 @@ int _nss_tcb_getspent_r(struct spwd *__result_buf,
 
 	do {
 		currpos = telldir(tcbdir);
+#ifdef GLIBC_2_24
+		saved_errno = errno;
```

Comment:
Where `errno` is restored?

Reply:
`errno` will be restored in line 133.  It keeps the same logic as with `readdir_r`, but takes in consideration that `readdir` will set errno on failure.

Reply:
I think we can mark this one as resolved, too?

Reply:
Yes, I think so.